docs/QA_MIGRATIONS_GUIDE.md

```diff
@@ -1,13 +1,31 @@
-### QA Migrations Guide
+## QA Migrations Guide
+
+### Upgrade
```

Comment:
Thanks for updating this guide. These steps are very comprehensive and provide more clarity.

Can we merge these steps with the steps in the `migration` section below to avoid duplication?

Reply:
Took a shot at this with fe9cca0831c3b715573b4fc3755b6489c1d19d6e